# Домашнее задание Семинара 2

### Задание 1. Вероятность того, что стрелок попадет в мишень, выстрелив один раз, равна 0.8. Стрелок выстрелил 100 раз. Найдите вероятность того, что стрелок попадет в цель ровно 85 раз.

In [6]:
import numpy as np
from math import factorial

### k=85, n = 100, p = 0.8, q = 0.2

In [8]:
def combinations (n,k):
    return np.math.factorial(n)//(np.math.factorial(k)*np.math.factorial(n-k))

In [11]:
result = combinations(100,85)

In [14]:
answer = pow(0.8,85)*pow(0.2,15)*result

In [17]:
round(answer,4)

0.0481

### Ответ: вероятность 4.81%

### Задание 2. Вероятность того, что лампочка перегорит в течение первого дня эксплуатации, равна 0.0004. В жилом комплексе после ремонта в один день включили 5000 новых лампочек. Какова вероятность, что ни одна из них не перегорит в первый день? Какова вероятность, что перегорят ровно две?

## p = 0.0004, n = 5000

In [39]:
lamda = 5000*0.0004

In [56]:
p_zero= (pow(2,0)/factorial(0))*np.math.exp(-2)

In [57]:
round(p_zero,4)

0.1353

In [58]:
p_two= (pow(2,2)/factorial(2))*np.math.exp(-2)

In [59]:
round(p_two,4)

0.2707

### Задание 3. Монету подбросили 144 раза. Какова вероятность, что орел выпадет ровно 70 раз?

### k=70, n = 144, p = 0.5, q = 0.5

In [19]:
result3 = combinations(144,70)

In [22]:
answer3 = pow(0.5,70)*pow(0.5,74)*result

In [23]:
round(answer3,4)

0.0628

### Ответ: вероятность 6.28%

## Задание 4. В первом ящике находится 10 мячей, из которых 7 - белые. Во втором ящике - 11 мячей, из которых 9 белых. Из каждого ящика вытаскивают случайным образом по два мяча. Какова вероятность того, что все мячи белые? Какова вероятность того, что ровно два мяча белые? Какова вероятность того, что хотя бы один мяч белый?

#### Все белые

In [34]:
p_w=combinations (7,2)/ combinations (10,2)*combinations (2,2)/ combinations (11,2)

In [35]:
p_w

0.008484848484848486

#### Два белые

In [24]:
p1 = combinations (7,2)/ combinations (10,2)* combinations (2,2)/ combinations (11,2)

In [25]:
p2 = combinations (3,2)/ combinations (10,2)* combinations (9,2)/ combinations (11,2)

In [26]:
p3 = (combinations (7,1)*combinations (3,1)/ combinations (10,2))* (combinations (9,1)*combinations (2,1)/combinations (11,2))

In [27]:
answer4 = p1+p2+p3

In [28]:
round(answer4,4)

0.2048

### Хотя бы один белый

In [36]:
p_ow = 1-combinations (3,2)/ combinations (10,2)* combinations (2,2)/ combinations (11,2)

In [37]:
round(p_ow,4)

0.9988